.github/workflows/update_dependencies_zulip.yml

```diff
@@ -14,6 +14,21 @@ jobs:
     if: ${{ github.event.workflow_run.conclusion == 'failure' }}
 
     steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+        with:
+          fetch-depth: 2  # Need previous commit for diff
+
+      - name: Set up Python
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        with:
+          python-version: '3.x'
+
+      - name: Parse dependency changes
+        id: parse_changes
+        run: |
+          echo "changes=$(./scripts/parse_lake_manifest_changes.py)" >> $GITHUB_OUTPUT
```

Comment:
How about storing the output of the script into a variable and then printing the variable in CI, to store it in the logs, before adding it to the output?

Reply:
Also, do multiline outputs work with `change=...`, instead of the `<<EOI...EOI` format?

Reply:
Done. And multiline outputs should Just Work (TM), iirc.